.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,76 @@
+---
+name: Bug report
+about: Create a report to help us reproduce and correct the bug
+title: ''
+labels: 'Bug: triage'
+assignees: ''
+
+---
+
+<!--
+Before submitting a bug, please make sure the issue hasn't been already
+addressed by searching through the past issues.
+
+If your issue is a usage question, please submit it in one of these other
+channels instead:
+- StackOverflow with the `fairlearn` tag:
+  https://stackoverflow.com/questions/tagged/fairlearn
+- Gitter: https://gitter.im/fairlearn/community#
+The issue tracker is used only to report bugs and feature requests. For
+questions, please use either of the above platforms. Most question issues are
+closed without an answer on this issue tracker. Thanks for your understanding.
+-->
+
+#### Describe the bug
+<!--
+A clear and concise description of what the bug is.
+-->
+
+#### Steps/Code to Reproduce
+<!--
+Please add a minimal example that we can reproduce the error by running the
+code. Be as succinct as possible, do not depend on external data. In short, we
+are going to copy-paste your code and we expect to get the same
+result as you.
+
+Example:
+```python
+from fairlearn.reductions import ExponentiatedGradient, DemographicParity
+from sklearn.linear_model import LinearRegression
+from tempeh.configurations import datasets
+
+dataset = datasets['adult_uci']
+X, _ = dataset.get_X()
+y, _ = dataset.get_y()
+sensitive_features = X[:, 7]
+mitigator = ExponentiatedGradient(LinearRegression(), DemographicParity())
+mitigator.fit(X, y, sensitive_features=sensitive_features)
+```
+If the code is too long, feel free to put it in a public gist and link
+it in the issue: https://gist.github.com
+-->
+
+```
+Sample code to reproduce the problem
+```
+
+#### Expected Results
+<!-- Example: No error is thrown. Please paste or describe the expected results.-->
+
+#### Actual Results
+<!-- Please paste or specifically describe the actual output or traceback. -->
+
+#### Screenshots
+<!-- If applicable, add screenshots to help explain your problem. -->
+
+#### Versions
+<!--
+Please provide the following information:
+- OS: [e.g. Windows]
+- Browser (if you're reporting a dashboard bug in jupyter): [e.g. Edge, Firefox, Chrome, Safari]
+- Python version: [e.g. 3.7.4]
+- Fairlearn version: [e.g. 0.4.5 or installed from master branch in editable mode]
+- version of Python packages: please run `pip freeze` and paste the output.
```

Comment:
The issue with `pip freeze` is that it doesn't list conda packages AFAIK. So you don't really get a good idea of what's installed. I find knowing the versions of just dependencies helps more.

In general, giving them a piece of code to run and paste the result lowers the barrier substantially.

Reply:
Fair point. We do still need to know if they have installed Fairlearn from pip, or if they checked out the repo, though.

Reply:
fairlearn version should give you that. The repo should always have a `dev[n]` version.

Reply:
That would be a change to how we're organising things right now. Currently, the repo has the last-released version. Which is not to say that we don't need to rethink our release process, as we've been mentioning in other discussions.

Reply:
moved away from pip freeze to show_versions. This `dev[n]` discussion is beyond the topic of the PR but I encourage having it anyway